### PR TITLE
WEB-133: Display tab for Coaches.

### PR DESCRIPTION
**Pull request by polesye**
@auraz please review
@den99f FYI.

### CODE REVIEW

File: lms/djangoapps/teacher_dashboard/tabs.py [hunk 1]
```diff
@@ -19,5 +21,29 @@ class TeacherDashboardTab(CourseTab):
...
+        Returns True when:
+            Teached dashboard feature is enabled
+            AND (
+                user has staff role
+                OR (
+                    CCX feature is enabled AND user has coach role
+                )
+            )
+        """
+        if not (user and settings.LABSTER_FEATURES.get('ENABLE_TEACHER_DASHBOARD', False)):
```
Comment (auraz) [inline lms/djangoapps/teacher_dashboard/tabs.py:34]: it was easier to read with is_feature_enabled

File: lms/djangoapps/teacher_dashboard/tabs.py [hunk 2]
```diff
@@ -19,5 +21,29 @@ class TeacherDashboardTab(CourseTab):
 
     @classmethod
     def is_enabled(cls, course, user=None):
-        is_feature_enabled = settings.LABSTER_FEATURES.get('ENABLE_TEACHER_DASHBOARD', False)
-        return bool(user and has_access(user, 'staff', course, course.id)) and is_feature_enabled
+        """
+        Returns True when:
```
Comment (auraz) [inline lms/djangoapps/teacher_dashboard/tabs.py:25]: User should be not none?

File: lms/djangoapps/teacher_dashboard/tabs.py (hunk 1 above)
Comment (polesye) [inline lms/djangoapps/teacher_dashboard/tabs.py:34]: > it was easier to read with is_feature_enabled

`ENABLE_TEACHER_DASHBOARD` is also pretty easy readable :)

File: lms/djangoapps/teacher_dashboard/tabs.py (hunk 2 above)
Comment (polesye) [inline lms/djangoapps/teacher_dashboard/tabs.py:25]: > User should be not none?

yep

Comment (polesye) [inline lms/djangoapps/teacher_dashboard/tabs.py:25]: > User should be not none?

@auraz Do you want me to add it to comments?

Comment (auraz) [inline lms/djangoapps/teacher_dashboard/tabs.py:25]: No

**Comment by auraz** [comment]
:+1:

**Pull request merged**